docs/source/app_developers_guide/docker.rst

```diff
@@ -0,0 +1,417 @@
+
+
+Transaction Processors
+======================
+
+The Docker Compose file (*sawtooth-default.yaml*) starts the following
+transaction processors automatically:
+
+* tp_config
+* tp_intkey_python
+* tp_xo_python
+
+These transaction processors handle transactions for the config, intkey, and XO transaction families, and are written in Python.
+
+
+Creating And Submitting Transactions
+====================================
+
+The **intkey** command is provided to create sample transactions of the intkey
+transaction type for testing purposes.
+
+This section guide you through the following tasks:
+
+1. Prepare a batch of intkey transactions that set the keys to random values.
+
+2. Generate *inc* (increment) and *dec* (decrement) transactions to apply to
+   the existing state stored in the blockchain.
+
+3. Submit these transactions to the validator.
+
+Run the following commands from the client container:
+
+.. code-block:: console
+
+  $ intkey create_batch
+  $ intkey load -f batches.intkey -U http://rest_api:8080
+
+You can observe the processing of the intkey transactions by observing the
+logging output of the intkey transaction processor and the validator, from the
+terminal window in which you ran the *docker-compose* command. A truncated
+example of the output:
+
+.. code-block:: console
+
+  tp_intkey_python_1  | [21:02:53.164 DEBUG    handler] Incrementing "VaUEPt" by 1
+  validator_1         | [21:02:53.169 DEBUG    interconnect] ServerThread receiving TP_STATE_SET_REQUEST message: 194 bytes
+  validator_1         | [21:02:53.171 DEBUG    tp_state_handlers] SET: ['1cf126d8a50604ea6ab1b82b33705fc3eeb7199f09ff2ccbc52016bbf33ade68dc23f5']
+  validator_1         | [21:02:53.172 DEBUG    interconnect] ServerThread sending TP_STATE_SET_RESPONSE to b'63cf2e2566714070'
+  validator_1         | [21:02:53.176 DEBUG    interconnect] ServerThread receiving TP_PROCESS_RESPONSE message: 69 bytes
+  validator_1         | [21:02:53.177 DEBUG    interconnect] message round trip: TP_PROCESS_RESPONSE 0.042026519775390625
+  validator_1         | [21:02:53.182 DEBUG    interconnect] ServerThread sending TP_PROCESS_REQUEST to b'63cf2e2566714070'
+  tp_intkey_python_1  | [21:02:53.185 DEBUG    core] received message of type: TP_PROCESS_REQUEST
+  validator_1         | [21:02:53.191 DEBUG    interconnect] ServerThread receiving TP_STATE_GET_REQUEST message: 177 bytes
+  validator_1         | [21:02:53.195 DEBUG    tp_state_handlers] GET: [('1cf126721fff0dc4ccb345fb145eb9e30cb7b046a7dd7b51bf7393998eb58d40df5f9a', b'\xa1fZeuYwh\x1a\x00\x019%')]
+  validator_1         | [21:02:53.200 DEBUG    interconnect] ServerThread sending TP_STATE_GET_RESPONSE to b'63cf2e2566714070'
+  tp_intkey_python_1  | [21:02:53.202 DEBUG    handler] Incrementing "ZeuYwh" by 1
+
+
+Config Transaction Family Usage
+===============================
+
+Sawtooth provides a :doc:`config transaction family
+<../transaction_family_specifications/config_transaction_family>` that stores on-
```

Comment:
There's one of these in the ubuntu doc at https://github.com/hyperledger/sawtooth-core/pull/546/files#diff-5f7497f925dbae7a0d4e62bcc9df1242R284. Since you're already in the docs, can you fix that one too?